src/arvcamera.c

```diff
@@ -2427,6 +2427,65 @@ arv_camera_is_binning_available (ArvCamera *camera, GError **error)
 	return horizontal && vertical;
 }
 
+/**
+ * arv_camera_get_representation_gain:
+ * @camera: a #ArvCamera
+ * @error: a #GError placeholder, %NULL to ignore
+ *
+ * Return: enum ArvGcRepresentation, -1 if not available.
+ *
+ * Since: 
+ */
+
+ArvGcRepresentation
+arv_camera_get_representation_gain (ArvCamera *camera, GError **error)
```

Comment:
Please rename this function to arv_camera_get_gain_representation()
Remove the error placeholder. Errors are for communication problems, not programming errors. get_representation comes directly from the xml data.

Reply:
updated in b9c1305b92d0322ec1c7f1ebef44a47d78db6a99